ot/deprecation.py

```diff
@@ -0,0 +1,103 @@
+"""
```

Comment:
Personally I think this short module is a bit overkill, maybe move the class and function to the utils.py module with the origin URL.

Reply:
Ok I do that